content/general/authentication.textile

```diff
@@ -139,22 +141,91 @@ ably.requestToken({ clientId: 'client@example.com' }, function(err, token) {
      } */
 });
 
-h3(#capabilities-explained). Capabilities and Token Security explained
+h4(#ably-jwt-process). Ably JWT is created by your servers and passed to clients
 
-"API keys":https://support.ably.io/solution/articles/3000030054-what-is-an-app-api-key, like "tokens":#tokens, have a set of capabilities assigned to them that specify which "operations":#capability-operations (such as subscribe or publish) can be performed on which channels. However, unlike tokens, API keys are long-lived, secret and typically not shared with un-trusted clients.
+It is possible to use a "JWT":https://jwt.io as a form of token for authentication with Ably, so long as it is structured appropriately, in what will be referred to as an *"Ably JWT":#ably-jwt*. t is possible for an Ably JWT to contain claims indicating its clientId, capabilities and expiry - in an analogous way to an "Ably Token":#tokens - and it is signed with an applicable Ably API key.
```

Comment:
Missing `I` in ". t is possible for an.."
clientId, capabilities, expiry are attribnutes, so style them as code please, i.e. `@clientId@, @capabilities@...`
Why an applicable Ably API key, why not just an Ably API key?